nextflow.config

```diff
@@ -91,6 +93,7 @@ profiles {
         params.enable_conda    = true
         conda.useMamba         = true
         docker.enabled         = false
+        conda.enabled          = true
```

Comment:
(note that this is because https://github.com/nf-core/tools/pull/1952 won't have been in the release version of the tools Oskar used)